inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -749,6 +750,7 @@ InferenceEngine::IExecutableNetworkInternal::Ptr Engine::ImportNetwork(std::istr
 
     auto execNetwork = std::make_shared<MKLDNNExecNetwork>(cnnnetwork, conf, extensionManager, weightsSharing);
 
+    SetExeNetworkInfo(execNetwork, cnnnetwork.getFunction());
```

Comment:
who will call it for other plugins? 
Can we create new inputs / outputs based on existing `GetInputsInfo()`, `GetOutputsInfo()` for import flow?

Reply:
let's create task on GNA, KMB, MYRIAD plugins